tailscale_compliance/tailscale.sp

```diff
@@ -0,0 +1,241 @@
+locals {
+  tailscale_common_tags = merge(local.tailscale_compliance__common_tags, {
+    tailscale = "true"
+  })
+}
+
+variable "api_key_id" {
+  type        = string
```

Comment:
Would the control(s) that use this still make sense if this was instead a list of strings? This would then allow me to check multiple tailnet keys with a single control run instead of needing to modify my variable each run